shepherd_bms/src/stateMachine.cpp

```diff
@@ -508,7 +508,7 @@ void balanceCells(AccumulatorData_t *bms_data)
 	segment.configureBalancing(balanceConfig);
 }
 
-void eepromLogFault(AccumulatorData_t *bmsdata, BMSFault_t fault)
+void StateMachine::eepromLogFault(AccumulatorData_t *bmsdata, BMSFault_t fault)
```

Comment:
Since we might use EEPROM for other logging (idk like notable data points, general logging of everything, etc idk) I think it would be a better practice to create a general framework for writing and reading data similar to a filesystem. I bet there are resources online on how to emulate directories and files in EEPROM, but essentially it is all just linked lists. Not sure if that's within the scope of this ticket but for scalability it would be very nice